pkg/apis/policy/validation/validation.go

```diff
@@ -39,6 +40,13 @@ import (
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 )
 
+const (
+	csiDriverNameRexpFmt    = `^[a-zA-Z0-9][-a-zA-Z0-9_.]{0,61}[a-zA-Z-0-9]$`
```

Comment:
This requires at least 2 letters - is that the correct spec?  Or is it something like 
 
`^[a-zA-Z0-9]([-a-zA-Z0-9_.]{0,61}[a-zA-Z-0-9])?$`